modules/microshift-stopping-lvms-csi-snapshot.adoc

```diff
@@ -0,0 +1,60 @@
+// Module included in the following assemblies:
+//
+// * microshift_storage/microshift-storage-plugin-overview.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-stopping-lvms-csi-snapshot_{context}"]
+== Stopping deployments that run CSI implementations
+
+User workloads with attached volumes must be manually stopped before disabling the CSI driver or CSI snapshot implementation pods. Use the following procedure to disable installation of one or both CSI components.
+
+[NOTE]
+====
+MicroShift will not delete CSI implementation pods. MicroShift must be configured to disable installation of one or both of the CSI pods during the start-up process.
```

Comment:
Attribute for "MicroShift"? {microshift-short}?

Active voice:

> You must configure MicroShift to disable installation of one or both of the CSI pods during the startup process.

![Screenshot from 2024-09-04 09-15-28](https://github.com/user-attachments/assets/0434bf68-ebe4-43dd-8432-fcea3b846f01)

Reply:
done